.github/workflows/unit_testing.yml

```diff
@@ -86,7 +86,7 @@ jobs:
 
   ios-unit-testing:
     needs: files-changed
-    if: ${{ inputs.force_ios || needs.files-changed.outputs.ios == 'true' || needs.files-changed.outputs.shared == 'true' }}
+    if: ${{ inputs.force_ios || needs.files-changed.outputs.ios == 'true' }}
```

Comment:
Also delete this condition (`|| needs.files-changed.outputs.shared == 'true'`) for `android-unit-testing`